agent/src/agent/pipeline/schema.py

```diff
@@ -14,8 +14,12 @@ def build(pipeline: Pipeline) -> dict:
             'action': 'fill',
             'fill': 'NULL'
         },
-        'dvpConfig': pipeline.dvp_config,
     }
+    if pipeline.dvp_config:  # set dvpConfig in schema only if it set in pipeline config
```

Comment:
if you want to warn about anodot setting default values when the empty config is provided, I would rather write specifically about it